index.js

```diff
@@ -572,18 +599,14 @@ class TuyaDevice extends EventEmitter {
   async toggle(property = '1') {
     property = property.toString();
 
-    try {
-      // Get status
-      const status = await this.get({dps: property});
+    // Get status
+    const status = await this.get({dps: property});
```

Comment:
Good catch, don't know why I put in a try/catch block there. 😛

Reply:
:+1: